### PR TITLE
feat: get reviews by status (expired, pending, active)

### PR DESCRIPTION
**Pull request by Extheoisah**

### CODE REVIEW

File: api/app/controllers/review.controller.ts [hunk 1]
```diff
@@ -251,6 +251,29 @@ export const getAllReviewsForAdmin = async (req: Request, res: Response) => {
...
 
+  if (status) {
+    const currentTime = new Date().getTime();
+    switch (status) {
+      case QUERY_REVIEW_STATUS.ACTIVE:
+        const activeCondition = buildIsActiveCondition(currentTime);
+        condition[Op.and as unknown as keyof typeof Op] = activeCondition;
+        break;
+
+      case "expired":
```
Comment (nully0x) [inline api/app/controllers/review.controller.ts:262]: I think the "expired" can be scoped to constant.

Comment (Extheoisah) [reply api/app/controllers/review.controller.ts:262]: I don't think that's necessary as we only use this here. Its not used anywhere else in the code.

**Review by nully0x — APPROVED** [review]
added new filtering by status

**Comment by kouloumos** [comment]
Active reviews are the ones in the queue?

**Pull request merged**